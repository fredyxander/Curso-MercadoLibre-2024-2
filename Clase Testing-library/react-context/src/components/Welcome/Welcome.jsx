export const Welcome = ({title})=>{
    return(
        <div className="welcome-page">
            <h1>{title}</h1>
            <h2>Clase React y Node</h2>
            <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Nobis explicabo maxime quisquam consequatur minus, amet, unde sed dolorum, rerum magnam atque accusamus dicta sunt deleniti fugiat recusandae deserunt sit culpa.</p>
        </div>
    )
}