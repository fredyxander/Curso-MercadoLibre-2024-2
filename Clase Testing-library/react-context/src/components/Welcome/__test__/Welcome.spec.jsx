import { render } from "@testing-library/react";
import { Welcome } from "../Welcome";

describe("Pruebas <Welcome/>", ()=>{
    test("Snapshot Welcome", ()=>{
        //renderiza el component
        const {asFragment} = render(<Welcome title="Bienvenidos"/>);
        expect(asFragment()).toMatchSnapshot();
    });
})