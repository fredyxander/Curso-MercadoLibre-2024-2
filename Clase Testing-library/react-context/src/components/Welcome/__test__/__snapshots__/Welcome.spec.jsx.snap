// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas <Welcome/> Snapshot Welcome 1`] = `
<DocumentFragment>
  <div
    class="welcome-page"
  >
    <h1>
      Bienvenidos
    </h1>
    <h2>
      Clase React y Node
    </h2>
    <p>
      Lorem ipsum, dolor sit amet consectetur adipisicing elit. Nobis explicabo maxime quisquam consequatur minus, amet, unde sed dolorum, rerum magnam atque accusamus dicta sunt deleniti fugiat recusandae deserunt sit culpa.
    </p>
  </div>
</DocumentFragment>
`;
